volontulo/frontend/scss/customization/_bootflat-panel.scss

```diff
@@ -98,10 +98,12 @@ $panel-background-color:        $white !default;
     > .panel-heading {
         @include border-left-radius(3px);
         @include border-right-radius(0);
+        padding: 10px 15px;
     }
     > .panel-body {
         @include border-left-radius(0);
         @include border-right-radius(3px);
         background: white;
+        padding: 10px 15px;
```

Comment:
that might be good idea, but the thing is... I'm not sure yet if this padding should be the same for both of these elements. 
Could change it now, who knows maybe it will stay as is